scripts/kube-init.sh

```diff
@@ -145,4 +158,4 @@ kubectl get services
 echo "Running tests..."
 set -x -e
 # Yield execution to venv command
-$*
+$*
```

Comment:
⛔ means that GitHub is expecting every file to end with one and only one carriage return (\n).